<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="serial_layout"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PelcoD" name="pelco_d">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Чтение данных</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item row="5" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Пресет помывки:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sb_addr_upu"/>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="sb_time_preset_washout"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="sb_time_washout"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Пресет временный:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Адрес камеры:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="sb_preset_washout"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Интервал дворника:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sb_time_interval"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Время помывки:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="sb_addr_cam"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Адрес модуля УПУ:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Время между помывками:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="sb_time_pause_washout"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cmd_31">
        <property name="text">
         <string>Чтение состояния (31)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cmd_34">
        <property name="text">
         <string>Чтение данных (34)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cmd_42">
        <property name="text">
         <string>Тест (42)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>75</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Запись данных</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sb_addr_upu_2"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Адрес модуля УПУ:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Адрес камеры:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Пресет помывки:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="sb_time_preset_washout_2"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Пресет временный:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Время помывки:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="sb_preset_washout_2"/>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="sb_addr_cam_2"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="sb_time_washout_2"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sb_time_interval_2"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Интервал дворника:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="sb_time_pause_washout_2"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Время между помывками:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cmd_36">
        <property name="text">
         <string>Запись данных (36)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cmd_37">
        <property name="text">
         <string>Запись данных (37)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PelcoD</class>
   <extends>QFrame</extends>
   <header>pelcod.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sb_addr_upu</tabstop>
  <tabstop>sb_time_interval</tabstop>
  <tabstop>sb_time_washout</tabstop>
  <tabstop>sb_time_pause_washout</tabstop>
  <tabstop>sb_addr_cam</tabstop>
  <tabstop>sb_preset_washout</tabstop>
  <tabstop>sb_time_preset_washout</tabstop>
  <tabstop>btn_cmd_31</tabstop>
  <tabstop>btn_cmd_34</tabstop>
  <tabstop>btn_cmd_42</tabstop>
  <tabstop>sb_addr_upu_2</tabstop>
  <tabstop>sb_time_interval_2</tabstop>
  <tabstop>sb_time_washout_2</tabstop>
  <tabstop>sb_time_pause_washout_2</tabstop>
  <tabstop>sb_addr_cam_2</tabstop>
  <tabstop>sb_preset_washout_2</tabstop>
  <tabstop>sb_time_preset_washout_2</tabstop>
  <tabstop>btn_cmd_36</tabstop>
  <tabstop>btn_cmd_37</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
